app/org/sagebionetworks/bridge/dynamodb/OptionLookup.java

```diff
@@ -35,5 +34,10 @@ public String get(String healthCode) {
     public SharingScope getSharingScope(String healthCode) {
         return SharingScope.valueOf(get(healthCode));
     }
+    
+    public Set<String> getDataGroups(String healthCode) {
+        String value = map.get(healthCode);
+        return BridgeUtils.commaListToSet(value);
+    }
```

Comment:
Is it worth adding a unit test for getDataGroups(), just to make sure we exercise the code?